src/index.ts

```diff
@@ -241,7 +241,7 @@ export default class Tooltip {
       return;
     }
 
-    const styles = require('../styles/main.css');
+    const styles = fetch('../styles/main.css');
```

Comment:
you don't need to manually append styles if you're using `cssInjectedByJsPlugin` plugin

Reply:
If do not change it, an error appears in the console and the styles are not applied

<img width="705" alt="image" src="https://github.com/codex-team/codex.tooltips/assets/46301523/ad6a6d82-35eb-4d55-afd7-dd7d602af141">